privaterelay/templates/profile.html

```diff
@@ -87,8 +87,21 @@
         <span class="dashboard-greeting-message">{% ftlmsg 'profile-label-welcome-html' email=request.user.email %}
       </div>
       {% endif %}
-      {% include "includes/banners.html" %}
 
+      <!-- Data Notification Banner -->
+      {% include "includes/banners/announcement-banner.html" %}
+      
+      <!-- Register Subdomain Banner -->
+      {% with request.user.profile_set.first as user_profile %}
+      {% if user_profile.has_premium and not user_profile.subdomain and settings.PREMIUM_ENABLED %}
+        {% include "includes/banners/subdomain-banner.html" %}
+      {% endif %}
+      {% endwith %}
+  
+      <!-- Upgrade, Firefox and Add On Banners -->
```

Comment:
These default banners (upgrade to premium/download Firefox/download the add on) are stored in this partial since these banners are consistent for premium/non-premium states (with the exception of upgrade being only for non-premium).